lib/active_admin_sidebar/activeadmin_views_pages_base.rb

```diff
@@ -9,6 +9,14 @@ def build_page_content
     end
   end
 
+  def build_sidebar
```

Comment:
this is the "magic" method

because of change here https://github.com/activeadmin/activeadmin/commit/bd34a9ef51003e204074a902f7fe14a53eabd585#diff-1ca3d5d936ee4939e354692f648b27e2R76